llm/llama.go

```diff
@@ -424,7 +424,7 @@ type PredictRequest struct {
 	TfsZ             float32         `json:"tfs_z,omitempty"`
 	TypicalP         float32         `json:"typical_p,omitempty"`
 	RepeatLastN      int             `json:"repeat_last_n,omitempty"`
-	Temperature      float32         `json:"temperature,omitempty"`
+	Temperature      float32         `json:"temperature"`
```

Comment:
Great catch! Could we update the rest of these to also remove `omitempty`? It seems that we set them all below so there shouldn't be any that are empty, except `logit_bias` and `n_probs`